### ISSUE TITLE
Failed dumping TiDB data using mydumper

### ISSUE THREAD

**Issue by aeoluswing** [pingcap/tidb#10935]
## Bug Report

1. What did you do?
If possible, provide a recipe for reproducing the error.

Dumping tidb data will make mydumper critical occasionally.

2. What did you expect to see?
Handle success.

3. What did you see instead?
```
** (mydumper:1): CRITICAL **: 17:17:58.896: Could not read data from ****.***: other error: request outdated
```

4. What version of TiDB are you using (`tidb-server -V` or run `select tidb_version();` on TiDB)?
version: 2.1.11

**Comment by aeoluswing** [comment]
Command like this for mydumper: 
```
$ mydumper --host=xxxx --database=xxxx--port=3306 --user=xxxx --password=xxxx --chunk-filesize=64 --threads=16 --skip-tz-utc -o /path/to/backup
```

**Comment by bb7133** [comment]
Thanks for reporting @aeoluswing 
Would you please take a look? @csuzhangxc

**Comment by shenli** [comment]
@aeoluswing Please check the system log like `dmesg`. I guess you meet OOM issue. It is a known issue for 2.1.11. Please upgrade to the 2.1.13.

**Comment by nullnotnil** [comment]
This issue was fixed in TiDB 2.1.13.

I am going to close this issue now, but please feel free to re-open if you have additional questions. Thanks!
